sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -62,6 +83,38 @@ public SmsClient createSmsClientWithAAD() {
         return smsClient;
     }
 
+    public SmsAsyncClient createAsyncClientUsingTokenCredential() {
+        TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
+        // You can find your endpoint and access key from your resource in the Azure Portal
+        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+
+        // Create an HttpClient builder of your choice and customize it
+        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+        SmsAsyncClient smsClient = new SmsClientBuilder().
+            endpoint(endpoint)
+            .credential(tokenCredential)
+            .httpClient(httpClient)
+            .buildAsyncClient();
+        return smsClient;
+
+    }
+
+    public SmsClient createSyncClientUsingTokenCredential() {
```

Comment:
Same here, I dont think we need to show this for both async and sync clients.
I would suggest picking one of the two to reduce duplication